language/Spanish/strings.po

```diff
@@ -2423,7 +2444,7 @@ msgstr "Elegir máquina nueva"
 #~ "Falta la máquina seleccionada o una o más de sus ROMs o imágenes CHD. Por "
 #~ "favor, elige otra máquina.\n"
 #~ "\n"
-#~ "Pulsa cualquier tecla para continuar."
+#~ "Pulsa cualquier tecla (menos ESC) para continuar."
```

Comment:
The strings in this section are from a previous version of MAME and no longer used (note that they're all preceded by `#` characters) - they're provided as reference to possibly help maintain terminology when translating updated strings.  There's no point modifying translations in this section.  You should remove this section after updating the translation.